code-challenges.js

```diff
@@ -9,60 +9,155 @@
 // Describe your process:
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+//const fruit1 = "apple"
+//const fruit2 = "banana"
 // Expected outcome: "banana"
 
 // Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+//const fruit3 = "cherry"
+//const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
+            //  Creating this function, I will utilize the Boolean primitive data type. I spent the most time on this code with a failed results.
+// I created a function using the provided variables. I created a const |boilPoint| to run everything in console.log.
+// The output reads |Cannot access 'temp1' before initialization|. I attempted different combinations and checked my syntax and have not been 
+//able to display the correct result. 
+
+
+
+
+
+    const boilPoint = (temp1) => {
+    if(temp1 < 212) {
+        return `${temp1} is below boiling point`
+    }
+    else if(temp2 > 212){
+        return `${temp2} is above boiling point`
+    }
+    else( temp3 === 212)  ;
+        return `${temp3} is at boiling point`
+    }
+
+console.log(boilPoint(temp1))
+console.log(boilPoint(temp2))
+console.log(boilPoint(temp3))
 
 const temp1 = 42
-// Expected output: "42 is below boiling point"
+  // Expected output: "42 is below boiling point"
 
 const temp2 = 350
 // Expected output: "350 is above boiling point"
 
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+        //To solve this, I begin with the accessor |.join| in order to combine the
```

Comment:
Don't forget to update your pseudocode if you find you need to change directions when you start coding.